testing/srt-test-live.cpp

```diff
@@ -791,6 +791,15 @@ int main( int argc, char** argv )
     alarm(0);
     end_time = time(0);
 
+    if (!src || !tar)
+    {
+        string tarstate = tar ? "CREATED" : "FAILED";
+        string srcstate = src ? "CREATED" : "FAILED";
```

Comment:
`const char*` might be better.

Reply:
This is test app, and I don't think about micro-optimizations in code like that, moreover, it's just to display an error before exiting.

Reply:
Yes, I agree. Still some `const` is better. Even if it is `const string`.